### PR TITLE
Automate tests for example scripts

### PR DESCRIPTION
**Pull request by Bultako**
This PR addresses issue https://github.com/gammapy/gammapy/issues/2158

It allows for testing the example scripts provided by `gammapy download` with a new `make test-scripts` declarative, as it is already the case when testing notebooks with `make test-nb`. A commented block is also added in `.travis.yml` in case it is decided to add these tests to CI.

### CODE REVIEW

File: gammapy/utils/scripts_test.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                return True
+    return False
+
+
+def script_test(path):
+    """Check if example Python script is broken."""
+    log.info("   ... EXECUTING {}".format(str(path)))
+
+    cmd = "python {}".format(path)
+    status, output = getstatusoutput(cmd)
```
Comment (cdeil) [inline gammapy/utils/scripts_test.py]: `subprocess.getstatusoutput` is only there for backwards compatibility, it's use is discouraged:
https://docs.python.org/3/library/subprocess.html#legacy-shell-invocation-functions

I was mostly using `subprocess.call` and `subprocess.check_call` so far:
https://docs.python.org/3/library/subprocess.html#subprocess.check_call

But it looks like in Python 3.5 `subprocess.run` was added and it's now the first / often recommended way to run subprocesses:
https://docs.python.org/3/library/subprocess.html#using-the-subprocess-module

@Bultako - could you try to use that here?

Comment (Bultako) [reply gammapy/utils/scripts_test.py]: I've made the modif.
It works well with `subprocess.run()`

**Comment by cdeil** [comment]
@Bultako - Could you uncomment the added travis-ci build?

Let me know when you're done here, and I'll merge it in later today.

**Comment by Bultako** [comment]
Done.
Travis-ci build uncommented.

**Comment by cdeil** [comment]
The `capture_output` isn't available on Python 3.5, build failed: https://travis-ci.org/gammapy/gammapy/jobs/538640197#L1198
We could rewrite using subprocess.Popen, but I think it's fine if that just runs on Python 3.6, so I'll switch to that now.

**Comment by cdeil** [comment]
@Bultako - Thanks!

**Pull request merged**